package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vscode-cpptools",
-  "version": "4.0.3",
+  "version": "5.0.0",
```

Comment:
We also need to update the `export enum Version` in `api.ts` and add a `v5` (which will be consumed by cpptools)